tests/draft2019-09/anchor.json

```diff
@@ -83,5 +77,53 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "Fragment syntax disallowed",
+        "schema": {
+            "$ref": "#foo",
+            "$defs": {
+                "A": {
+                    "$anchor": "#foo",
+                    "type": "integer"
+                }
+            }
+        },
+        "tests": [
+            {
+                "data": 1,
+                "description": "match",
+                "valid": false
+            },
+            {
+                "data": "a",
+                "description": "mismatch",
+                "valid": false
+            }
+        ]
+    },
+    {
+        "description": "Invalid characters disallowed",
+        "schema": {
+            "$ref": "#/a/b",
```

Comment:
This doesn't look like a valid schema from what I can see in the metaschema.

Reply:
Ah, this is a place where the schema needs to be correct. This was my old way of doing it. I'll fix it.

Reply:
Turns out I'm putting these in #354.